scripts/check_duplicate_api_locators.sh

```diff
@@ -18,7 +18,7 @@ function main() {
 
     cd "${ROOT_DIR}"
 
-    local duplicateLocators=$(grep -R 'Error("-' "${API_DIR}" | sed 's/^.*"\(-[0-9]\{3,\}\)".*$/\1/' | sort | uniq -c | grep -ZvP '^\s+1\s+' | sed 's/^\s*\([0-9]\+\)\s\+\(-[0-9]\{3,\}\)$/\2/')
+    local duplicateLocators=$(grep -R 'Error("-' "${API_DIR}" | sed 's/^.*"\(-[0-9]\{3,\}\)".*$/\1/' | sort | uniq -c | grep -EZv '^[[:space:]]*1[[:space:]]+' | sed 's/^\s*\([0-9]\+\)\s\+\(-[0-9]\{3,\}\)$/\2/')
```

Comment:
I think plus would be better here than star (similar reasons with the other comment).